contracts/SlashIndicator.sol

```diff
@@ -197,6 +206,19 @@ contract SlashIndicator is ISlashIndicator,System,IParamSubscriber, IApplication
     emit indicatorCleaned();
   }
 
+  function downtimeSlash(address validator, uint256 count) external override onlyValidatorContract {
+    _downtimeSlash(validator, count);
+  }
+
+  function _downtimeSlash(address validator, uint256 count) internal {
+    try IStakeHub(STAKE_HUB_ADDR).downtimeSlash(validator) {
+    } catch Error(string memory reason) {
```

Comment:
Becase we already catch error here, I think the stakehub contract do not need to catch error any more.

Reply:
We don't catch error in stakeHub